pkg/controller/workload/workload_processor.go

```diff
@@ -105,6 +107,7 @@ func (p *Processor) processWorkloadResponse(rsp *service_discovery_v3.DeltaDisco
 	if err != nil {
 		log.Error(err)
 	}
+	kmeshbpf.SetStartType(kmeshbpf.Normal)
```

Comment:
`kmeshbpf.SetStartType` has been called in `compareWorkloadAndServiceWithHashName`, do you need remove that？

Reply:
you mean handleRemoveAddressDuringRestart？ kmeshbpf.SetStartType has been removed